app/models/user.rb

```diff
@@ -15,4 +15,29 @@ class User < RedisStore
 
   validates :uuid, presence: true
   validates :email, presence: true
+
+  def claims
+    Claim.fetch_all(vaafi_headers)
+  end
+
+  private
+
+  def vaafi_headers
+    {
+      "va_eauth_pnidtype" => "SSN",
+      "va_eauth_csid" => "DSLogon",
+      "va_eauth_firstName" => "Jane",
+      "va_eauth_lastName" => "Doe",
+      "va_eauth_authenticationauthority" => "eauth",
+      "iv-user" => "dslogoneauthuser",
+      "va_eauth_emailAddress" => "jane.doe@va.gov",
+      "va_eauth_birthdate" => "1999-10-09T08:06:12-04:00",
+      "va_eauth_pid" => "123456789",
+      "va_eauth_issueinstant" => "2015-04-17T14:52:48Z",
+      "va_eauth_dodedipnid" => "1105051936",
+      "va_eauth_middleName" => "A",
+      "va_eauth_authenticationmethod" => "DSLogon",
+      "va_eauth_assurancelevel" => "2"
+    }
+  end
```

Comment:
this is just test data, once we have authentication this method would construct these headers using data from the session. we can make it a constant for now if we want, but i also wanted it to be clear that this will be different per User instance